css/style.css

```
@@ -15,5 +14,178 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.header {
+  display: flex;
+  justify-content: space-around;
+}
+#logo {
+  width: 100px;
+  height: 50px;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+.navbar ul li {
+  padding: 10px;
+}
+
+.list-nav li a {
+  text-decoration: none;
+  color: black;
+  opacity: 0.6;
+}
+.list-nav li a:hover {
```

```suggestion
.list-nav li a:focus,
.list-nav li a:hover {
```
We should also add a `:focus` for users who use the `Tab` key to jump to the link, so it also changes colour when that happens